docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
       - VPN_IPSEC_PSK=${L2TP_IPSEC_PSK}
       - VPN_USER_CREDENTIAL_LIST=${L2TP_USER_LIST}
   openconnect:
-    image: tommylau/ocserv:latest
+    image: quay.io/aminvakil/ocserv:latest
```

Comment:
why we have to use this ?

Reply:
`tommylau/ocserv` has not been maintained for quite some time. You can look at the commits on top of tommylau repository (86 so far) and see if it's worth.